src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -37,6 +76,36 @@ function getDrupalValue(arr) {
   return null;
 }
 
+/**
+ * A very specific helper function that expects to receive an
+ * object and an imageStyle string
+ * @param {object}
+ * @return {object}
+ */
+function getImageCrop(obj, imageStyle = null) {
+  if (imageStyle !== null) {
+    const imageObj = Object.assign({}, obj);
+    const image = mediaImageStyles.find(({ style }) => style === imageStyle);
+    // If imageStyle is not found, it will return the raw obj
+    if (!image) {
+      throw new Error(
+        `${imageStyle} imageStyle was not found. mediaImageStyles available are ${mediaImageStyles
+          .map(s => s.style)
+          .join(', ')}.`,
+      );
+    }
```

Comment:
I'm a lot more comfortable with this, thanks!

I _really_ want TypeScript enums, but :man_shrugging: Runtime validation will have to do.